packages/panzer/core/src/Panzer_PauseToAttach.hpp

```diff
@@ -74,7 +75,7 @@ namespace panzer {
 
   inline void pauseToAttach()
   {
-    pauseToAttach(MPI_COMM_WORLD);
```

Comment:
I think this actually needs to be `MPI_COMM_WORLD` to make sense.

Reply:
Fixed